modules/agent-install-about-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// * installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc
+// re-use of applicable content from disconnected install mirroring
+
+:_content-type: CONCEPT
+[id="agent-install-about-mirroring-for-disconnected-registry_{context}"]
+= About mirroring the {product-title} image repository for a disconnected registry
+
+To use mirror images for a disconnected installation with the Agent-based Installer, you must modify the `install-config.yaml` file.
+
+You can mirror the release image by using the output of either the `oc adm release mirror` or `oc mirror` command.
+This is dependent on which command you used to set up the mirror registry. The following example shows the output of the `oc adm release mirror` command.
+
+
+[source,terminal]
+----
+$ oc adm release mirror
+----
+
+.Example output
+
+[source,terminal]
+----
+To use the new mirrored repository to install, add the following
+section to the install-config.yaml:
+
+imageContentSources:
+
+mirrors:
+virthost.ostest.test.metalkube.org:5000/localimages/local-release-image
+source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+mirrors:
+virthost.ostest.test.metalkube.org:5000/localimages/local-release-image
+source: registry.ci.openshift.org/ocp/release
+----
+Use the output of the command to update the `imageContentSources` field in the `install-config.yaml` file. Additionally, add the certificate file used for the mirror registry to the `additionalTrustBundle` field.
+
+.Example
+
+[source,yaml]
+----
+  additionalTrustBundle: |
+    -----BEGIN CERTIFICATE-----
+    ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+    -----END CERTIFICATE-----
+----
+
+The value must be the contents of the certificate file that you used for your mirror registry. The certificate file can be an existing, trusted certificate authority, or the self-signed certificate that you generated for the mirror registry.
+
+If you are using the optional method of the ZTP manifests, there are two mirror configuration files: `registries.conf` and `ca- bundle.crt` . These files must be added to the `mirror/directory` path to add the mirror configuration in the agent ISO image.
+
+You can create the `registries.conf` file from the output of either the `oc adm release mirror` or `oc mirror` command. The format of the `/etc/containers/registries.conf` file has changed recently. It is now version 2 and in TOML format.
```

Comment:
I think that _recently_ will get stale. Was version 2 introduced in OpenShift 4.12? Will old registries.conf files that use the version 1 format and YAML still work?

Reply:
@bfournie could you please help confirm the version? Thanks

Reply:
Confirmed that it is version 2  with @bfournie.  In 4.11 also version 2 is used  ref https://docs.openshift.com/container-platform/4.11/openshift_images/image-configuration.html

Reply:
Additional ref >> https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#version-1-format---deprecated

Reply:
Version 1 has been deprecated, it is still supported but it does not support using registry mirrors so that version won't be useful for this support, see https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#version-1-format---deprecated

Reply:
I agree with removing the word recently.